rpc/beefy/beefy.go

```diff
@@ -0,0 +1,31 @@
+// Go Substrate RPC Client (GSRPC) provides APIs and types around Polkadot and any Substrate-based chain RPC calls
+//
+// Copyright 2021 Snowfork
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package beefy
+
+import (
+	"github.com/centrifuge/go-substrate-rpc-client/v4/client"
+)
+
+// Beefy exposes methods for retrieval of chain data
+type Beefy struct {
+	client client.Client
+}
+
+// NewBeefy creates a new Chain struct
+func NewBeefy(cl client.Client) *Beefy {
+	return &Beefy{cl}
+}
```

Comment:
Is there an advantage to doing `NewBeffy(client)` vs `&Beefy{client}`? If not,  I'd drop this redundant construct. If there is, you can rename it to just `New` so that its usage becomes `beefy.New(cl)`.